visualizations/frontend/other/TTable.vue

```diff
@@ -771,17 +772,33 @@ export default {
         column.default_value
       );
       if (cellValue && typeof cellValue === "object") {
-        if (cellValue.rendered) cellValue = cellValue.rendered;
+        if (cellValue.value){
+            cellValue = cellValue.value;
+        } 
         else cellValue = "";
       }
       return cellValue;
     },
     getCellValue(row, column) {
-      const cellValue = this.getRawCellValue(row, column);
+      const cellValue = this.getRenderedCellValue(row, column);
       return column.format
         ? column.format(cellValue, row.originalRow)
         : cellValue;
     },
+    getRenderedCellValue(row, column){
+        let cellValue = _.get(
+        row.originalRow,
+        column.property,
+        column.default_value
+      );
+      if (cellValue && typeof cellValue === "object") {
+        if (cellValue.rendered){
+            cellValue = cellValue.rendered;
+        } 
+        else cellValue = "";
```

Comment:
Should this fallback to `cellValue.value`? I don't think there's ever a case where `rendered` and `value` aren't both present, so this whole method might be more complex than necessary.

Reply:
Awesome, if both rendered and value are always present then this can absolutely be simplified. updated. 👍